content/kit/servo-assembly/GPIO.md

```diff
@@ -0,0 +1,14 @@
+---
+title: GPIO
+---
+
+The arduino allows you to connect your kit to your own electronics. It has 14 digital I/O pins, and 4 analogue. The analogue pins can read an analogue signal from 0 to 5V. The board also has a couple of ground pins, as well as some pins fixed at 3.3V and 5V output.
```

Comment:
I'd recommend saying 6 analogue pins here rather than 4, since the you go on to say immediately below that two are reserved for the servo shield.